xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-ui/src/main/resources/Main/SolrSearchMacros.xml

```diff
@@ -42,13 +42,11 @@
 #set ($rangePattern = $regextool.compile('^[\[{](.+) TO (.+)[\]}]$'))
 #set ($wildcardPattern = $regextool.compile('^\(.*\*.*\)$'))
 
-#macro (displaySearchForm)
+#macro (_displaySearchFormBegin)
```

Comment:
Is it really necessary to rename this Velocity macro? This macro is used by [at least one contrib extension](https://github.com/xwiki-contrib/application-interactive-maps/blob/a58b1cf5349fef3b2093a47aca6baafeda2c9fcf/application-interactive-maps-ui/src/main/resources/Maps/Code/CommonMacros.xml#L402) that breaks due to this change. I'm not sure if we shouldn't consider these macros API as the search is frequently customized.